### PR TITLE
Add get and set for `Env`

### PR DESCRIPTION
**Pull request by edwin0cheng**
This PR add three things :

1. Add `get` and `set` in `Env`.
2. Implement fixture meta for `with_single_file`. 
3. Add `env` meta in fixture.

### CODE REVIEW

File: crates/test_utils/src/lib.rs [hunk 1]
```diff
@@ -202,6 +202,14 @@ pub fn parse_fixture(fixture: &str) -> Vec<FixtureEntry> {
     res
 }
 
+/// Same as `parse_fixture`, except it allow empty fixture
+pub fn parse_single_fixture(fixture: &str) -> Option<FixtureEntry> {
+    if !fixture.lines().any(|it| it.trim_start().starts_with("//-")) {
+        return None;
+    }
+    parse_fixture(fixture).into_iter().nth(0)
```
Comment (matklad) [inline crates/test_utils/src/lib.rs]: We probably should check that it's the *only* fixture?

Comment (edwin0cheng) [reply crates/test_utils/src/lib.rs]: Fixed.

**Comment by edwin0cheng** [comment]
rebased. 

bors retry

**Pull request merged**